packages/next/src/build/webpack/config/blocks/css/index.ts

```diff
@@ -568,6 +568,18 @@ export const css = curry(async function css(
           chunkFilename: ctx.isProduction
             ? 'static/css/[contenthash].css'
             : 'static/css/[name].css',
+          insert: function (link) {
+            var found = false
+            for (var i = 0; i < document.head.children.length; i++) {
+              var current = document.head.children.item(i)
+              if (current.href == link.href) {
+                found = true
+                current.insertAdjacentElement(link)
+                break
+              }
+            }
+            if (!found) document.head.appendChild(linkTag)
```

Comment:
I assume it should be `link` here, not `linkTag`? Even with that, this fix doesn't seem to work.

Reply:
This is what I get for copy/paste from my node_modules. One sec.